src/sentry/tasks/store.py

```diff
@@ -100,22 +100,25 @@ def should_demote_symbolication(project_id):
             "project_id": project_id,
         },
     )
-    return not never_lowpri and always_lowpri
 
+    if never_lowpri:
+        return False
+    elif always_lowpri:
+        return True
+    else:
+        return realtime_metrics.is_lpq_project(project_id)
```

Comment:
we said we'd run this in dark-launch for a while, i.e. not to actually move to the queue.  I wonder what the best way to go about this is, should we log it, increase a counter metric?  Maybe the latter is best, we could keep that counter even after the dark launch.

The second part is wither we want to be able to control it via a sentry option or only via code changes.  Maybe a (boolean) sentry option would be nice as it could be switched off again in an emergency.

What do you think?

Also, could this be re-based agains the right branch?  currently it's rather confusing as to what is actually in this commit.

Reply:
I added a sentry option.